dagger-common/src/test/java/io/odpf/dagger/common/serde/typehandler/PrimitiveTypeHandlerTest.java

```diff
@@ -17,104 +17,104 @@
         Descriptors.Descriptor descriptor = TestBookingLogMessage.getDescriptor();
         Descriptors.FieldDescriptor fieldDescriptor = descriptor.findFieldByName("cancel_reason_id");
-        PrimitiveProtoHandler primitiveProtoHandler = new PrimitiveProtoHandler(fieldDescriptor);
+        PrimitiveTypeHandler primitiveTypeHandler = new PrimitiveTypeHandler(fieldDescriptor);
 
-        assertEquals(1, primitiveProtoHandler.transformFromPostProcessor("1"));
+        assertEquals(1, primitiveTypeHandler.transformFromPostProcessor("1"));
     }
 
     @Test
     public void shouldReturnStringValueForStringTypeFieldDescriptorIfStringIsPassedForPostProcessorTransform() {
         Descriptors.Descriptor descriptor = TestBookingLogMessage.getDescriptor();
         Descriptors.FieldDescriptor stringFieldDescriptor = descriptor.findFieldByName("order_number");
-        PrimitiveProtoHandler primitiveProtoHandler = new PrimitiveProtoHandler(stringFieldDescriptor);
+        PrimitiveTypeHandler primitiveTypeHandler = new PrimitiveTypeHandler(stringFieldDescriptor);
 
-        assertEquals("123", primitiveProtoHandler.transformFromPostProcessor("123"));
+        assertEquals("123", primitiveTypeHandler.transformFromPostProcessor("123"));
     }
 
     @Test
     public void shouldReturnStringValueForStringTypeFieldDescriptorIfStringIsNotPassedForPostProcessorTransform() {
         Descriptors.Descriptor descriptor = TestBookingLogMessage.getDescriptor();
         Descriptors.FieldDescriptor stringFieldDescriptor = descriptor.findFieldByName("order_number");
-        PrimitiveProtoHandler primitiveProtoHandler = new PrimitiveProtoHandler(stringFieldDescriptor);
+        PrimitiveTypeHandler primitiveTypeHandler = new PrimitiveTypeHandler(stringFieldDescriptor);
 
-        assertEquals("123", primitiveProtoHandler.transformFromPostProcessor(123));
+        assertEquals("123", primitiveTypeHandler.transformFromPostProcessor(123));
     }
 
     @Test
     public void shouldThrowInvalidDataTypeExceptionInCaseOfTypeMismatchForPostProcessorTransform() {
         Descriptors.FieldDescriptor floatFieldDescriptor = TestBookingLogMessage.getDescriptor().findFieldByName("total_customer_discount");
-        PrimitiveProtoHandler primitiveProtoHandler = new PrimitiveProtoHandler(floatFieldDescriptor);
+        PrimitiveTypeHandler primitiveTypeHandler = new PrimitiveTypeHandler(floatFieldDescriptor);
 
         InvalidDataTypeException exception = Assert.assertThrows(InvalidDataTypeException.class,
-                () -> primitiveProtoHandler.transformFromPostProcessor("stringValue"));
+                () -> primitiveTypeHandler.transformFromPostProcessor("stringValue"));
         assertEquals("type mismatch of field: total_customer_discount, expecting FLOAT type, actual type class java.lang.String", exception.getMessage());
     }
 
     @Test
     public void shouldReturnSameValueForTransformFromKafka() {
```

Comment:
need to change name of the test from `TransformForKafka` to `FromProto`. Needs to be done in multiple places

Reply:
will fix shortly

Reply:
@gauravsinghania Fixed via commit https://github.com/odpf/dagger/pull/140/commits/256abb26e3d094371e6b99c88712dcafcb8cd30f